addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -247,7 +249,7 @@
                                     <div class="dropdown-menu" t-att-id="'navToogleTagGroup%s' % tag_group.id">
                                         <t t-foreach="tag_group.tag_ids" t-as="tag">
                                             <a rel="nofollow" t-att-class="'dropdown-item %s' % ('active' if tag in search_tags else '')"
-                                                t-att-href="'/slides/all?%s' % keep_query('*', **{'channel_tag_group_id_%s' % tag_group.id: tag.id if tag not in search_tags else False})"
+                                                t-att-href="'/slides/all?%s' % (keep_query('*', tags=str((search_tags - tag).ids)) if tag in search_tags else keep_query('*', tags=str((tag | search_tags).ids)))"
```

Comment:
Suggestion:

`keep_query('*', tags=str(( (search_tags - tag) if tag in search_tags else (tag | search_tags) ).ids))`